ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_replica_promotion.py
```

Comment:
You should run only TestHiddenReplicaPromotion and increase the timeout to 7200.